documentation/modules/proc-configuring-kafka-bridge-tracing.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-config.adoc
+
+[id='proc-configuring-kafka-bridge-tracing-{context}']
+= Configuring distributed tracing
+
+[role="_abstract"]
+Enable distributed tracing to trace messages consumed and produced by the Kafka Bridge, and HTTP requests from client applications.
+
+Properties to enable tracing are present in the `application.properties` file.
+To enable distributed tracing, do the following:
+
+* Uncomment the required tracing properties in the properties file.
```

Comment:
I would not say in that way.
I would more focusing on the `bridge.tracing` property that should be set in the file.
Something like "set the bridge.tracing property value enabling the tracing you want to use. Possible values are jaeger and opentelemetry".
The fact that we are providing the `application.properties` file with commented lines is for simplicity but I would not use it for explaining how it works.
Wdyt?

Reply:
Works for me. I was trying to keep things very simple in the intro. But I think this is better